.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -33,7 +33,9 @@ as any relevant images for UI changes._
 
 ### UI accessibility concerns?
 
-_If your PR includes UI changes, please replace this line with details on how accessibility is impacted and tested. For more info, check out the [Forem Accessibility Docs](https://docs.forem.com/frontend/accessibility)._
+_If your PR includes UI changes, please replace this line with details on how
```

Comment:
@narender2031, if you merge latest from master, this file and several others will no longer be part of your PR. See https://github.com/forem/forem/pull/9741#issuecomment-732482975

I'd do it myself, but I do not appear to have access to push to your branch.

Reply:
I will do that.